model/resource/k8s.yaml

```diff
@@ -45,6 +45,8 @@ groups:
     attributes:
       - ref: k8s.container.name
       - ref: k8s.container.restart_count
+      - ref: k8s.container.status.last_terminated_reason
+        requirement_level: opt_in
```

Comment:
any reason for it to be opt-in? does it have sensitive info, is it expensive to collect, is it too verbose? 
If not, I'd consider `recommended` so users can leverage it by default and instrumentations wouldn't need to add config options.

Reply:
I am good with `recommended` as I don't feel high cardinality (or in this case the potential of high cardinality) should prevent an attribute from being recommended. I think in practice there will be a pretty small set of values.

Reply:
> it might cause high cardinality

Is this true 🤔? Do we have an indicator of how many different values this attribute could come with?

I would expect what @TylerHelmuth already mentioned:
> in practice there will be a pretty small set of values